docs/analysis-tools/variant-distance-matrix.md

```diff
@@ -0,0 +1,40 @@
+**Variant Distance Matrix**
+===============
+About the Variant Distance Matrix Tool
+--------------------------
+The Variant Distance Matrix tool calculates the disimilarity between variant sets in the database and stores the results back into the database. It only calculates the distances between variant sets of the same species and includes sanity checks to ensure that duplicate or improper calculations do not occur. This allows for other tools, most notably the SimilarVariantSet tool, to quickly determine relationships between variant sets. 
+
+The Variant Distance Matrix tool makes one assumption when calculates the distance between variant sets: the genome is static within the same species. In other words, the variant distance matrix tool assumes that any two variant sets for which the distance is calculated for have the same reference genome. This allows the tool to assume that the distance between two variant sets is only a function of the differences between the variants in the variant sets and not any other outside variables. Since both variant sets have the same static genome, the distance is calculated using the following formula:
```

Comment:
remove "for" from "which the distance is calculated for have the same reference genome."